docs/modules/integrations/pages/github-actions/run-appium-script.adoc

```diff
@@ -0,0 +1,15 @@
+= Run Appium script using GitHub Actions
+:navtitle: Run Appium script
+
+Learn how to utilize Kobiton automation testing with Appium script and GitHub Actions.

```

Comment:
Suggestion: 
Learn how to run an Appium script on Kobiton with GitHub Actions.

Reply:
@Hikkigaya can you work on this?

Reply:
Sure

Reply:
Updated